library/ziti.c

```diff
@@ -571,3 +578,22 @@ static const ziti_version sdk_version = {
 const ziti_version *NF_get_version() {
     return &sdk_version;
 }
+
+static void grim_reaper(uv_prepare_t *p) {
+    nf_context nf = p->data;
+
+    int count = 0;
+    const char *url;
+    ziti_channel_t *ch;
+    MODEL_MAP_FOREACH(url, ch, nf->channels) {
+        nf_connection conn = LIST_FIRST(&ch->connections);
+        while(conn != NULL) {
+            nf_connection try_close = conn;
```

Comment:
this should have some kind of log associated with it shouldn't it? This will be a really hard bug to find with no trace of where the connection went if there's a bug?

Reply:
added log msg in conn_close_internal